sass/style.css

```diff
@@ -6,16 +6,27 @@ html {
 body {
   background: #fff;
   box-sizing: border-box;
-  color: #222;
-  font: normal normal normal 1rem/1.65 'Roboto', 'Helvetica', sans-serif;
+  color: #333;
+  font: normal normal normal 1rem/1.65 'Open Sans', 'Helvetica', sans-serif;
   height: 100%;
   margin: 0;
   padding: 0;
   text-rendering: optimizeLegibility;
+  font-size: 14px;
+  line-height: 1.55;
```

Comment:
You have 1rem/1.65 in the font property. Can combine here.

Reply:
Done!